.github/workflows/pkgci_shark_ai.yml

```diff
@@ -81,3 +93,7 @@ jobs:
         run: |
           source ${VENV_DIR}/bin/activate
           pytest -v -s app_tests/integration_tests/llm/shortfin --log-cli-level=INFO
+
+      - name: Clean up repo to make next checkout faster
+        run: |
+          git clean -ffdx
```

Comment:
This should be part of the runner configuration, not the job.

https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/running-scripts-before-or-after-a-job

Reply:
Gotcha. Going to remove this and collect this into a new issue and ask Sai for help.